docs/source/apis/Products.rst

```diff
@@ -0,0 +1,412 @@
+
+   2. Combine these prices into a :py:class:`PriceToEstimateFees <mws.models.products.PriceToEstimateFees>` instance:
+
+      .. code-block:: python
+
+          from mws.models.products import PriceToEstimateFees
+
+          my_product_price = PriceToEstimateFees(listing_price=my_price, shipping=my_shipping)
+
+      For the JP market only, this price to estimate fees may optionally include
+      :py:class:`Points <mws.models.products.Points>`.
+
+   3. Use the ``PriceToEstimateFees`` instance along with other data to construct the final
+      ``FeesEstimateRequest`` instance:
+
+      .. code-block:: python
+
+          from mws.models.products import FeesEstimateRequest
+
+          estimate_request = FeesEstimateRequest(
+              marketplace_id=my_market,
+              id_type="ASIN",  # either 'ASIN' or 'SKU', indicating the type of the `id_value` argument:
+              id_value="B07QR73T66",
+              price_to_estimate_fees=my_product_price,  # your `PriceToEstimateFees` instance
+              is_amazon_fulfilled=False,
+              identifier="request001",  # a unique identifier of your choosing
+          )
+
+.. autoclass:: mws.models.products.PriceToEstimateFees
+
+   Accepts instances of :py:class:`MoneyType <mws.models.products.MoneyType>` for its ``listing_price`` and
+   ``shipping``, and optionally accepts a :py:class:`Points <mws.models.products.Points>` instance
+   to denote a points value (in JP region only).
+
+.. autoclass:: mws.models.products.MoneyType
+
+   .. rubric:: Example
+   .. code-block:: python
+
+      from mws.models.products import MoneyType, CurrencyCode
+
+      my_money = MoneyType(amount=3.50, currency_code=CurrencyCode.USD)
+
+.. autoclass:: mws.models.products.Points
+
+   Points are expressed in terms of a ``points_number`` and a ``monetary_value`` for those points, the latter of which
+   must be an instance of :py:class:`MoneyType <mws.models.products.MoneyType>`.
+
+   .. rubric:: Example:
+   .. code-block:: python
+
+      from mws.models.products import Points, MoneyType, CurrencyCode
+
+      # A monetary value of 2000 Japanese yen
+      monetary_value = MoneyType(amount=2000.0, currency_code=CurrencyCode.JPY)
+
+      # Now assign the points like so:
+      points = Points(points_number=35, monetary_value=monetary_value)
+
+   When used in a request, `points` will be converted to a set of parameters like so:
+
+   .. code-block:: python
+
+      print(points.to_params())
+      # {'PointsNumber': 35, 'PointsMonetaryValue.Amount': 2000.0, 'PointsMonetaryValue.CurrencyCode': <CurrencyCode.JPY: ('JPY', 'Japanese yen')>}
+
+   .. note:: You will see the ``PointsMonetaryValue.CurrencyCode`` element remains an instance of Enum at this stage.
+      When used in a request, it is automatically "cleaned" to its parameterized value, ``'JPY'``.
+
+      Passing the string literal ``'JPY'`` as the ``MoneyType.currency_code`` argument is also accepted.
+
+Enums
+=====
```

Comment:
There may be only one Enum in this entire section, but if ever there's a need for another, this is where they'll go.